lib/templates/updateDevice.json

```diff
@@ -200,6 +200,14 @@
       "description": "Free form array of data to be appended to the target entity",
       "type": "array"
     },
+    "timestamp": {
+      "description": "Timestamp",
+      "type": "boolean"
+    },
+    "expressionLanguage": {
+      "description": "Expression language used to apply expressions for this device",
+      "type": "string"
+    },
```

Comment:
```suggestion
```

Field no longer supported.

(Maybe appeared here because we forget remove it in the config grup equivalente template?)